OpenRA.Mods.RA/Traits/Infiltration/InfiltrateForActor.cs

```diff
@@ -13,18 +13,18 @@
 
 namespace OpenRA.Mods.RA.Traits
 {
-	class InfiltrateForSupportPowerInfo : ITraitInfo
+	class InfiltrateForActorInfo : ITraitInfo
```

Comment:
1) Sure. First fix the implementation. Upgrade rules are usually the last thing to be done.
2) No need to run this just yet. Also you can upgrade the official mods and maps by hand if you wish. We'll walk you through it when you get to it.